src/lib/options/default-options/default-object-conversion-options.ts

```diff
@@ -7,5 +7,7 @@ export const DEFAULT_OBJECT_CONVERSION_OPTIONS: ObjectConversionOptions = {
   delimiter: Delimiter.CAMEL,
   verbose: false,
   fileName: 'my-icons',
-  objectName: 'icons'
+  objectName: 'icons',
+  generateType: true,
+  typeName: 'MyType'
```

Comment:
How about setting the default to `MyIconType`?

Reply:
We should also apply this default in the `commanline-object-options.collector.ts`. This file sets all the defaults for configurations that is read from a `.rc` config file

Reply:
Fixed!
I did not pay attention that by mistake used `DEFAULT_CONST_CONVERSION_OPTIONS` where it was not meant to be

Reply:
Hi @cakeinpanic thx a lot. I checked the PR again and I think there is still one last thing missing. I think we also need to apply the default options in the `config-file-object-options.collector.ts`. Something like this.

`````

  if (!configOptions.generateType) {
    configOptions.generateType = DEFAULT_OBJECT_CONVERSION_OPTIONS.generateType;
    Logger.verboseInfo(`No 'generateType' property provided, "${DEFAULT_OBJECT_CONVERSION_OPTIONS.generateType}" will be used`);
  }

  if (!configOptions.typeName) {
    configOptions.generateType = DEFAULT_OBJECT_CONVERSION_OPTIONS.typeName;
    Logger.verboseInfo(`No 'typeName' property provided, "${DEFAULT_OBJECT_CONVERSION_OPTIONS.typeName}" will be used`);
  }
`````

Reply:
@kreuzerk added them, also tackled readme

Reply:
Awesome 👍  Did you push the README? I don't see in in the change list

Reply:
pushed now